.eslintrc.js

```diff
@@ -276,12 +283,6 @@ module.exports = {
           { assertionStyle: 'as' },
         ],
 
-        // Prefer interfaces over type literals (type T = { ... }).
-        '@typescript-eslint/consistent-type-definitions': [
```

Comment:
It turns out that interfaces and type literals are not completely equivalent, and there are use cases for type literals over interfaces. This rule does not account for those use cases.

See https://github.com/microsoft/TypeScript/issues/13573.